src/status_im2/contexts/chat/messages/list/style.cljs

```diff
@@ -18,45 +16,24 @@
 (def list-container
   {:padding-vertical 16})
 
-(defn header-container
-  [show? theme]
-  {:display          (if show? :flex :none)
-   :background-color (colors/theme-colors colors/white colors/neutral-100 theme)
-   :top              (- overscroll-cover-height)
-   :margin-bottom    (- overscroll-cover-height)})
-
-(defn header-cover
-  [cover-bg-color theme]
-  {:flex             1
-   :height           (+ overscroll-cover-height cover-height)
-   :background-color (colors/theme-colors
-                      (colors/custom-color cover-bg-color 50 20)
-                      (colors/custom-color cover-bg-color 50 40)
-                      theme)})
-
 (defn header-bottom-part
-  [animation theme]
+  [border-radius theme]
   (reanimated/apply-animations-to-style
-   {:border-top-right-radius animation
-    :border-top-left-radius  animation}
-   {:top              -16
-    :margin-bottom    -16
-    :padding-bottom   24
-    :background-color (colors/theme-colors colors/white colors/neutral-95 theme)}))
-
-(def header-avatar
-  {:top               header-avatar-top-offset
-   :margin-horizontal 20
-   :margin-bottom     header-avatar-top-offset})
+   {:border-top-left-radius  border-radius
+    :border-top-right-radius border-radius}
+   {:background-color   (colors/theme-colors colors/white colors/neutral-95 theme)
+    :padding-horizontal 20
+    :margin-top         (+ (safe-area/get-top)
+                           messages.constants/top-bar-height
+                           messages.constants/header-container-top-margin)}))
 
 (defn header-image
-  [scale-animation top-margin-animation side-margin-animation]
+  [scale-animation top left]
   (reanimated/apply-animations-to-style
-   {:transform     [{:scale scale-animation}]
-    :margin-top    top-margin-animation
-    :margin-left   side-margin-animation
-    :margin-bottom side-margin-animation}
-   {:align-items :flex-start}))
+   {:transform [{:scale scale-animation}]
+    :top       top
+    :left      left}
```

Comment:
I did similar changes in my PR here, but you can overwrite them, as yours looks smoother 👍

Reply:
It's just that removing this adds back other issues, since the margin animation changes the content-size, breaking some of the behavior from `on-content-size-change`. I would rather keep it there and help with the merge conflicts caused by this. What do you think?